results/intfloat__multilingual-e5-small/JAQKETRetrieval.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dataset_revision": "e4af6c73182bebb41d94cb336846e5a452454ea7",
+  "mteb_dataset_name": "JAQKETRetrieval",
+  "mteb_version": "1.10.15",
+  "test": {
+    "evaluation_time": 837.92,
+    "map_at_1": 1.0,
+    "map_at_10": 1.0,
+    "map_at_100": 1.0,
+    "map_at_1000": 1.0,
+    "map_at_20": 1.0,
+    "map_at_3": 1.0,
+    "map_at_5": 1.0,
+    "mrr_at_1": 1.0,
+    "mrr_at_10": 1.0,
+    "mrr_at_100": 1.0,
+    "mrr_at_1000": 1.0,
+    "mrr_at_20": 1.0,
+    "mrr_at_3": 1.0,
+    "mrr_at_5": 1.0,
+    "ndcg_at_1": 1.0,
+    "ndcg_at_10": 1.0,
```

Comment:
perfect score seems a bit odd.

Reply:
Hmm is the task too easy? E.g. due to contamination?

Reply:
Hello, and thank you for your work!
Our evaluation result of `intfloat/multilingual-e5-small` on JAQKET is not as perfect as yours.
Our results on JAQKET evaluation split are
```json
{
    "accuracy@1": 0.35577889447236183,
    "accuracy@3": 0.5447236180904522,
    "accuracy@5": 0.6010050251256281,
    "accuracy@10": 0.6592964824120603,
    "ndcg@10": 0.5093087534845081,
    "mrr@10": 0.4610217755443884
}
```
Our results on JAQKET test split are
```json
{
    "accuracy@1": 0.3510531594784353,
    "accuracy@3": 0.5205616850551655,
    "accuracy@5": 0.5787362086258776,
    "accuracy@10": 0.6559679037111334,
    "ndcg@10": 0.49966509556428645,
    "mrr@10": 0.45011781375873633
}
```

Reply:
I checked the results of other datasets, and found that our results are numerically very close with yours. So I believe if there's something wrong, it's about this dataset only.

Reply:
Any idea why you might have different datasets?

Reply:
I noticed @awinml removed title information from corpus just now. That's also what I considered as the reason of the perfect results. Does everything go well after removing it?

Reply:
@lsz05 Thanks! Yes, on digging deep into the implementations of the retrieval evaluators, I saw that [JMTEB only uses the `text`](https://github.com/sbintuitions/JMTEB/blob/main/src/jmteb/evaluators/retrieval/evaluator.py#L80) for evaluation, whereas in [MTEB we concatenate the `Title` and `Text`](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/evaluation/evaluators/RetrievalEvaluator.py#L355). Hence decided to drop the `Title` for `JAQKET`.

The results after removing it are not perfect anymore. The NDCG values match, but the MRR still is different, and we get perfect recall at all levels.

Reply:
> I will run it on the "dev" split as well and compare.

@awinml how did this end up?

Reply:
@awinml I am just adding another ping here, in case you missed it. Understandable if you have other priorities

Reply:
@KennethEnevoldsen I tested the "dev" split as well, but was not able to replicate the results above. Unfortunately, I cannot prioritize more work on this currently, so have excluded the `JAQKET` dataset from this PR. Should be good to merge now.

Reply:
Thanks for taking the time @awinml